escrow/scripts/deploy.js

```diff
@@ -0,0 +1,47 @@
+const { Contract } = require("ethers");
+const { calcEthereumTransactionParams } = require("@acala-network/eth-providers");
+const { ACA } = require("@acala-network/contracts/utils/Address");
+
+const TokenContract = require("@acala-network/contracts/build/contracts/Token.json");
+
+const txFeePerGas = '199999946752';
+const storageByteDeposit = '100000000000000';
+
+const ESCROW_AMOUNT = 1_000_000_000; // 0.001 ACA
+
+async function main() {
+  const ethParams = calcEthereumTransactionParams({
+    gasLimit: '21000010',
+    validUntil: '360001',
+    storageLimit: '640010',
+    txFeePerGas,
+    storageByteDeposit
+  });
+
+  const [deployer, requestor, provider] = await ethers.getSigners();
+
+  console.log("Deploying contracts with the account:", deployer.address);
+
+  deployerAddress = await deployer.getAddress();
+  requestorAddress = await requestor.getAddress();
+  providerAddress = await provider.getAddress();
+
+  acaInstance = new Contract(ACA, TokenContract.abi, requestor);
+
+  const Escrow = await ethers.getContractFactory('Escrow');
+  const escrow = await Escrow.deploy(acaInstance.address, ESCROW_AMOUNT, requestorAddress, providerAddress,
+    {
+      gasPrice: ethParams.txGasPrice,
+      gasLimit: ethParams.txGasLimit,
+    }
+  );
+
+  console.log("Escrow address:", escrow.address);
```

Comment:
I think I can add some other information about the contract here, maybe all the parameters passed to its constructor?

Reply:
You could return the information about the escrow that was initiated or any other info that you think might make sense. 🤔